src/scandeval/dataset_configs.py

```diff
@@ -688,6 +688,37 @@ def get_dataset_config(dataset_name: str) -> DatasetConfig:
     unofficial=True,
 )
 
+ICE_EC_CONFIG = DatasetConfig(
+    name="ice-ec",
+    pretty_name="the Icelandic Error Corpus (a subset of the dataset)",
```

Comment:
Just to make it consistent with the other dataset configs:

```suggestion
    pretty_name="the truncated version of the Icelandic Error Corpus",
```